commerce-media-api/build.gradle

```diff
@@ -1,7 +1,10 @@
 dependencies {
 	compileOnly group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.1.0"
 	compileOnly group: "com.liferay", name: "com.liferay.osgi.util", version: "4.0.0"
+	compileOnly group: "com.liferay", name: "com.liferay.portal.configuration.metatype.api", version: "1.0.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "3.5.0"
 	compileOnly group: "javax.servlet", name: "javax.servlet-api", version: "3.0.1"
 	compileOnly group: "org.osgi", name: "org.osgi.core", version: "6.0.0"
+	compileOnly project(":apps:commerce:commerce-api")
```

Comment:
@alecsloan  We can't add this dependency because we are implicitly creating a dependency between commerce-api and commerce-product-x modules